### ISSUE TITLE
The resource `newrelic_browser_application` doesn't return the appId

### ISSUE THREAD

**Issue by Rtwena** [newrelic/terraform-provider-newrelic#2577]
The resource `newrelic_browser_application` doesn't return the appId. Which makes it difficult to later use the browser application, as I cannot add the appId to the outputs. Which is required for the newrelic JS script injection.

### Affected Resource(s)
- `newrelic_browser_application` https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/browser_application

### Terraform Configuration
```
    newrelic = {
      source  = "newrelic/newrelic"
      version = "3.30.0"
    }
```

### Actual Behavior
The browser appId is not available when after the resource has been created

### Expected Behavior
The browsr appId should be available after creating the resource, so that it can be used as an "output"

### Steps to Reproduce

1. create resource "newrelic_browser_application"

**Comment by pranav-new-relic** [comment]
Hi @Rtwena, thank you for reporting this; looks like this attribute has been missed out on. I'll bring this to the notice of the team, and we shall prioritise this based on our current task queue. Thanks!

**Comment by pranav-new-relic** [comment]
Hi @Rtwena, the change to facilitate this has been released with v3.34.0 of the Terraform Provider just sometime ago. I've validated the change with this new version, and it now returns the `application_id` of the browser application.

<img width="526" alt="image" src="https://github.com/newrelic/terraform-provider-newrelic/assets/127438038/0472c526-59ae-466d-a8b9-1830201aa58c">

Closing this issue - thanks for reporting this!
